src/parseSync.js

```diff
@@ -114,9 +114,25 @@ function spawnServer(opts) {
     };
   }
 
+  let rubyPlugins;
+  if (opts.rubyPlugins !== "") {
```

Comment:
Seems like this is setting it up such that if you specify _any_ plugins, it's going to not follow your preferences for `singleQuote` and `trailingComma`, which I don't think we want. Instead, we should split up the list here and see if `plugin/single_quotes` is present, and act accordingly. That way folks can still pass in their own custom plugins if they want.

Reply:
that's a great point - updated. let me know if that's more in line with what you had in mind